src/logic/http/response.ts

```diff
@@ -0,0 +1,34 @@
+import { IHttpServerComponent } from '@well-known-components/interfaces'
+
+export class HttpError extends Error {
+  constructor(message: string, public code: number) {
+    super(message)
+
+    // Set the prototype explicitly.
+    Object.setPrototypeOf(this, HttpError.prototype)
+  }
+}
+
+export async function json(
```

Comment:
maybe we can rename this to a something a bit more clear/verbose, what do you think about `toJSONResponse` or just `asJSON`

I say this because when I read `json({})` [here](https://github.com/decentraland/nft-server/pull/20/files#diff-f5e23d398d56faaad1a265ed540242d85c438872cf407d96bba4d396094e879eR25) I wasn't completely sure about what the function did before reading the implementation, to me it was doing a `JSON.parse()`